gslabModel/.gitignore

```diff
@@ -0,0 +1,4 @@
+.Rhistory
```

Comment:
Good catch. It's created when I create the package. I think it makes sense to merge it with the .gitignore at the root.